sasl_bind.c

```diff
@@ -0,0 +1,21 @@
+#include <ldap.h>
+#include "sasl_bind.h"
+
+static int
+sasl_trivial_interact (LDAP * ld, unsigned flags, void *defaults,
+                       void *sasl_interact)
+{
+  (void) ld;
+  (void) flags;
+  (void) defaults;
+  (void) sasl_interact;
+  return LDAP_SUCCESS;
+}
+
+int
+trivial_external_sasl_bind (LDAP * ld)
+{
+  return ldap_sasl_interactive_bind_s (ld, NULL, "EXTERNAL", NULL, NULL,
```

Comment:
1. man page suggests `ldap_sasl_interactive_bind_s()`
2. I tried `ldap_sasl_bind_s()` and got exception like "LDAPException LdapSaslBindInProgress(14): SASL bind in progress (SASL(0): successful result: )"